Sources/Theme.php

```diff
@@ -2730,11 +2749,23 @@ protected function loadVariant(): void
 		Utils::$context['theme_variant_url'] = '';
 
 		if (!empty($this->settings['theme_variants'])) {
+			// Add the default variant
+			$this->settings['theme_variants'] = array_unique(array_merge(['default'], $this->settings['theme_variants']));
+
```

Comment:
Likewise, not sure of this either

Reply:
One thought I have is if we ever need to care for spaces? Seems it has never been an issue and it's obvious enough when it doesn't work due to a space in the name of the variant.

And to add more context to "default". From my experience and investigation, "index_default.css" is almost never been used or included at all, it's just means to an end, a way to return to the default colors.